riak/transports/tcp/transport.py

```diff
@@ -78,7 +78,16 @@ def _get_codec(self, msg_code):
     # FeatureDetection API
     def _server_version(self):
         server_info = self.get_server_info()
-        return server_info['server_version']
+        ver = server_info['server_version']
+        (maj, min, patch) = [int(v) for v in ver.split('.')]
+        if maj == 0:
+            import datetime
+            now = datetime.datetime.now()
+            if now.year == 2016:
+                # GH-471 As of 20160509 Riak TS OSS 1.3.0 returns '0.8.0' as
+                # the version string.
+                return '2.1.1'
```

Comment:
Actually, right now in `riak_ee` the version is correct - `2.1.1`. It's only in Riak TS OSS that it comes back `0.8.0`.